docs/api/api_alternative.md

```diff
@@ -0,0 +1,289 @@
+
+## Scheduling
+
+### RTOS
+
+The Mbed OS RTOS capabilities include managing objects such as threads, synchronization objects and timers. It also provides interfaces for attaching an application-specific idle hook function, reads the OS tick count and implements functionality to report RTOS errors.
+
+| API | Full profile | Bare metal profile |
+| - | - | - |
+| [Thread](../apis/rtos-apis.html) | &#10004; | |
+| [ThisThread](../apis/thisthread.html) | &#10004; | &#10004;  |
+| [Mutex](../apis/mutex.html) | &#10004; | &#10004; |
+| [Semaphore](../apis/semaphore.html) | &#10004; | &#10004;  |
+| [Queue](../apis/queue.html) | &#10004; | |
+| [Mail](../apis/mail.html) | &#10004; | |
+| [EventFlags](../apis/eventflags.html) | &#10004; | &#10004; |
+| [Conditionvariable](../apis/conditionvariable.html) | &#10004; | |
+| [Kernel interface functions](../apis/kernel-interface-functions.html) | &#10004; | `get_ms_count` only |
+
+### Event handling
+
+| API | Full profile | Bare metal profile |
+| - | - | - |
+| [Event](../apis/event.html) | &#10004; | &#10003; (can be manually enabled) |
+| [EventQueue](../apis/eventqueue.html) | &#10004; | &#10003; (can be manually enabled) |
+| [UserAllocatedEvent](../apis/userallocatedevent.html) | &#10004; | &#10003; (can be manually enabled) |
+
+## Drivers
+
+### Input/Output drivers
+
+Input/Output APIs include analog and digital inputs and outputs on development boards, as well as digital interfaces, which allow your board to interface with a computer or external devices.
+
+| API | Full profile | Bare metal profile |
+| - | - | - |
+| [AnalogIn](../apis/i-o-apis.html) | &#10004; | &#10004; |
+| [AnalogOut](../apis/analogout.html) | &#10004; | &#10004; |
+| [DigitalIn](../apis/digitalin.html) | &#10004; | &#10004; |
+| [DigitalOut](../apis/digitalout.html) | &#10004; | &#10004; |
+| [DigitalInOut](../apis/digitalinout.html) | &#10004; | &#10004; |
+| [BusIn](../apis/busin.html) | &#10004; | &#10004; |
+| [BusOut](../apis/busout.html) | &#10004; | &#10004; |
+| [BusInOut](../apis/businout.html) | &#10004; | &#10004; |
+| [PortIn](../apis/portin.html) | &#10004; | &#10004;  |
+| [PortOut](../apis/portout.html) | &#10004; | &#10004;  |
+| [PortInOut](../apis/portinout.html) | &#10004; | &#10004;  |
+| [PwmOut](../apis/pwmout.html) | &#10004; | &#10004;  |
+| [InterruptIn](../apis/interruptin.html) | &#10004; | &#10004;  |
+
+### USB drivers
+
+The Mbed OS classes providing USB peripheral functionality, also known as USB components, inherit from USBDevice and provide specific USB peripherial functionality.
+
+| API | Full profile | Bare metal profile |
+| - | - | - |
+| [USBSerial](../apis/usb-apis.html) | &#10004; | &#10004;  |
+| [USBAudio](../apis/usbaudio.html) | &#10004; | &#10004;  |
+| [USBHID](../apis/ubshid.html) | &#10004; | &#10004;  |
+| [USBCDC](../apis/usbcdc.html) | &#10004; | &#10004;  |
+| [USBCDC_ECM](../apis/usbcdc-ecm.html) | &#10004; | &#10004;  |
```

Comment:
This is currently not supported in bare metal. We have a task to port it. It should be done by next week.